wgpu/_classes.py

```diff
@@ -2024,23 +2024,27 @@ def messages(self):
 
 
 class GPUQuerySet(GPUObjectBase):
-    """TODO"""
-
-    # IDL: undefined destroy();
-    def destroy(self):
-        """Destroy the queryset."""
-        raise NotImplementedError()
+    def __init__(self, label, internal, type, count):
+        super().__init__(label, internal, None)
+        self._label = label
```

Comment:
All objects created by the device have a reference to the device, and label is already stored in `self._label` by the `super().__init__`.
```suggestion
    def __init__(self, label, internal, device, type, count):
        super().__init__(label, internal, device)
```